spec/factory_bot/sequence_spec.rb

```diff
@@ -43,7 +44,9 @@
       end
     end
 
-    its(:names) { should eq [:test, :alias, :other] }
+    it "has the expected names as its names" do
+      expect(subject.names).to eq [:test, :alias, :other]
```

Comment:
The names are still a bit of a mystery guest (GitHub wasn't showing me thing lines directly above this at first, so I wasn't immediately sure where they came from). If you want to address that in this PR, you could have the subject for the shared example but then avoid using it for this particular spec.

Reply:
Good catch, fixed in latest commit.